Microsoft.Azure.CosmosRepository/src/Pagination/LimitedResultRequest.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) IEvangelist. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Azure.CosmosRepository.Pagination
+{
+    /// <inheritdoc />
+    public class LimitedResultRequest : ILimitedResultRequest
+    {
+        /// <summary>
+        /// Default value: 10.
+        /// </summary>
+        public static int DefaultMaxResultCount { get; set; } = 10;
```

Comment:
Because it is a global setting, if a consumer wants to return a customize count, they can modify the DefaultMaxResultCount  value when starting application,

Reply:
Is `10` a reasonable default? Why not `50`, or `100`?